src/js/stars.js

```diff
@@ -0,0 +1,36 @@
+const emptyStar = `<svg class="star" width="18" height="18" viewBox="0 0 18 18" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
puszczam to z zastrzeżeniem porównania/przejrzenia z najnowszymi gwiazdkami Daniela które ładnie działają teraz